packages/contracts-bedrock/scripts/Config.sol

```diff
@@ -15,15 +14,14 @@ library Config {
     ///         written to disk after doing a deployment.
     function deploymentOutfile() internal view returns (string memory _env) {
         _env = vm.envOr(
-            "DEPLOYMENT_OUTFILE", string.concat(vm.projectRoot(), "/deployments/", _getDeploymentContext(), "/.deploy")
+            "DEPLOYMENT_OUTFILE", string.concat(vm.projectRoot(), "/deployments/", vm.toString(block.chainid), "/deploy.json")
```

Comment:
I think this needs to be the chain name rather than id?

Reply:
Not sure if there is a generic way to get the chain name into this context. The artifacts are not committed into the repo. Its assumed that they are moved into superchain ops after a deployment